hat/conf/play.conf

```diff
@@ -83,13 +82,18 @@ play {
       allowed = [".hubofallthings.net", ".hubat.net", ${?HAT_DOMAIN}]
     }
 
-    enabled = []
+    enabled = [
+        "com.github.stijndehaes.playprometheusfilters.filters.StatusCounterFilter",
+        "com.github.stijndehaes.playprometheusfilters.filters.StatusAndRouteLatencyFilter",
+        "org.hatdex.hat.utils.LoggingFilter",
+        "play.filters.hosts.AllowedHostsFilter",
+        "play.filters.cors.CORSFilter",
+        "play.filters.gzip.GzipFilter",
+        "org.hatdex.hat.utils.TLSFilter"
```

Comment:
we should ditch that custom built filter in favour of the Play one (which can be switched on/off: `play.filters.https.RedirectHttpsFilter`)

Reply:
sure i dont know enough to comment on that but i dont think that is in the scope of this PR. Can you create a jira ticket in the backlog